hc/api/tests/test_list_checks.py

```diff
@@ -53,4 +87,10 @@ def test_it_shows_only_users_checks(self):
         for check in data["checks"]:
             self.assertNotEqual(check["name"], "Bob 1")
 
-    ### Test that it accepts an api_key in the request
+        r = self.get
```

Comment:
`self.get` should have `()` . That line should not be there altogether. we have fixed it